docs/protocol/agents/validators.mdx

```diff
@@ -0,0 +1,27 @@
+# Validators
+
+Validators are responsible for observing the [Mailbox](../mailbox.mdx) contract and, if required, signing its merkle root to facilitating message transmission to remote chains.
+
+Unlike many other protocols, Hyperlane does **not** have an enshrined validator set. Anyone is free to run a validator and help contribute to securing interchain messages.
```

Comment:
The last sentence is a bit misleading because the validator set registered on an ISM _is_ permissioned. People can run their own validators only as long as their receiver contract specifies a multisig ISM that does include that validator's address